### DISCUSSION TITLE
how to install aiogram

### DISCUSSION THREAD

**Discussion by Muzaffar-Xodjayev** [Muzaffar-Xadjayev/rus-uzb-translator-bot#5] [Q&A] [answered]
I'm going to write bot with aiogram but I don't know it

**Comment by Muzaffar-Xodjayev** [comment, marked as answer]
> I'm going to write bot with aiogram but I don't know it

I have many suggestion for you:
firstly you should visit documentation site

**Comment by salko-ua** [comment]
site: https://docs.aiogram.dev/en/latest/install.html
